spec/dummy/db/structure.sql

```diff
@@ -3962,3 +3965,5 @@ INSERT INTO schema_migrations (version) VALUES ('20141211200012');
 
 INSERT INTO schema_migrations (version) VALUES ('20141217171816');
 
+INSERT INTO schema_migrations (version) VALUES ('20150604000000');
+
```

Comment:
I personally do not think that structure.sql should be in the repo at all, since it gets generated by migration anyway. Leaving it in just puts us at risk for having inaccurate DDL here and confusing people.